src/Classes/Item.lua

```diff
@@ -827,15 +831,26 @@ function ItemClass:ParseRaw(raw, rarity, highQuality)
 		if not self.affixes then 
 			self.crafted = false
 		elseif self.rarity == "MAGIC" then
-			self.affixLimit = 2
+			if self.prefixes.limit or self.suffixes.limit then
+				self.prefixes.limit = m_max((self.prefixes.limit or 0) + 1, 0)
+				self.suffixes.limit = m_max((self.suffixes.limit or 0) + 1, 0)
+				self.affixLimit = self.prefixes.limit + self.suffixes.limit
+			else
```

Comment:
This isn't always true, you can make magic Simplex/Focused Amulets by using the split beast. If the base after a split has a prefix and a suffix it will still be magic

Reply:
Yes but you cant craft them into that state, the only way to achieve that is via split beast which ignores the affix limits

You can still make a magic item that has those affixes in PoB, just not as a crafted item, also I dont think most people will use a magic simplex/focused and thats only as an intermediate crafting step